WordPress/Classes/ViewRelated/NewGutenberg/NewGutenbergViewController.swift

```diff
@@ -68,10 +68,12 @@ class NewGutenbergViewController: UIViewController, PostEditor, PublishingEditor
     // MARK: - GutenbergKit
 
     private let editorViewController: GutenbergKit.EditorViewController
-    private weak var autosaveTimer: Timer?
+
+    lazy var autosaver = Autosaver() {
+        self.performAutoSave()
+    }
```

Comment:
Reinstated the use of `Autosaver` as it is a far more robust implementation, including triggering persistence if a number of changes threshold is crossed.